### PR TITLE
Adding normalization for parameter and current values

### PR DESCRIPTION
**Pull request by jeremyciak**
#### Pull Request (PR) description
Adding normalization for parameter and current values

#### This Pull Request (PR) fixes the following issues
#82

#### Task list
- [x] Added an entry under the Unreleased section of file CHANGELOG.md.
      Entry should say what was changed, and how that affects users (if applicable).
- [x] Resource documentation added/updated in README.md.
- [x] Resource parameter descriptions added/updated in README.md, schema.mof
      and comment-based help.
- [x] Comment-based help added/updated.
- [x] Localization strings added/updated in all localization files as appropriate.
- [x] Examples appropriately added/updated.
- [x] Unit tests added/updated. See [DSC Community Testing Guidelines](https://dsccommunity.org/guidelines/testing-guidelines).
- [x] Integration tests added/updated (where possible). See [DSC Community Testing Guidelines](https://dsccommunity.org/guidelines/testing-guidelines).
- [x] New/changed code adheres to [DSC Community Style Guidelines](https://dsccommunity.org/styleguidelines).

---
This change is [Reviewable](https://reviewable.io/reviews/dsccommunity/xremotedesktopsessionhost/83)

### CODE REVIEW

**Comment by danielboth** [comment]
Hi @jeremyciak, sorry for the delay in response. I had a look at this PR and remembered about Test-DscParameterState, a function that is included in the DscRsource.Common module (see here: https://github.com/dsccommunity/DscResource.Common/blob/master/source/Public/Test-DscParameterState.ps1). I think that might be the way to fix this, but I did not have time to test integrating that module yet, would you be up for that?

As an intermediate step we could also merge your code in it's current form, although I would prefer the previous. Let me know your thoughts.

**Comment by jeremyciak** [comment]
@danielboth, I believe I have a functional implementation of the `Test-DscParameterState` function, which of course has set the foundation to implement this and other functions from the `DscResource.Common` module across more DSC resources in this `xRemoteDesktopSessionHost` module.

**Comment by danielboth** [comment]
Hi @jeremyciak, thanks for this work, looks good at first sight! I'll do my best to do a full review by tomorrow.

**Comment by jeremyciak** [comment]
@danielboth, alright thanks! I also have a lot of other fixes that I would love to implement so if all goes well here you can expect more Issues and PR's to come through. I'm curious how does one become an official **Contributor** for this repository?

File: tests/Unit/MSFT_xRDSessionCollectionConfiguration.tests.ps1 [hunk 1]
```diff
@@ -259,7 +259,7 @@ try
                 [pscustomobject]@{
                     CollectionName           = 'TestCollection'
                     IncludeFolderPath        = $null
-                    ExcludeFolderPath        = $null
+                    ExcludeFolderPath        = @('c:\temp\foo', 'c:\temp\bar')
```
Comment (danielboth) [inline tests/Unit/MSFT_xRDSessionCollectionConfiguration.tests.ps1:262]: To ensure this property is compared in the test, Test-TargetResource in the test should be called with the -ExcludeFolderPath parameter (on the call here: https://github.com/dsccommunity/xRemoteDesktopSessionHost/blob/07a6a435114159d362d32183f8b5cf3458cfceda/tests/Unit/MSFT_xRDSessionCollectionConfiguration.tests.ps1#L295)

File: CHANGELOG.MD [hunk 2]
```diff
@@ -27,6 +27,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Changes to tests
   - Pin tests to use Pester v4 instead of latest (v5)
 
+### Fixed
```
Comment (danielboth) [inline CHANGELOG.MD:30]: Remove ### Fixed and just put this change under "Changed", like all the other fixes.

**Comment by danielboth** [comment]
Hi @jeremyciak, just some minor things, if you can change / update those we're ready for merge.

You become an official contributor the first time you merge a PR, so once this one is approved & merged.

**Comment by jeremyciak** [comment]
@danielboth, I suppose the question I was really meaning to ask was how to become a **Reviewer** and not just a **Contributor**

**Comment by jeremyciak** [comment]
@danielboth, how do I join this DSC community call? Also, does this PR need to be merged to master?

**Comment by danielboth** [comment]
Hi @jeremyciak, yes this PR needs to be merged to master. After it gets merged, we'll do an automatic release of the module with a preview tag (e.g. 2.0.0-preview0004). You should be able to merge this PR now, so please go ahead and do that.

On the DSC community, you can get more information here: https://dsccommunity.org/community_calls/. Also, for the schedule of the calls, please follow @dsccommunity on twitter.

**Comment by jeremyciak** [comment]
@danielboth, I don't believe I have write access to this repository so I am unable to merge my PR:

image

**Comment by danielboth** [comment]
Hi @jeremyciak, got it, I thought merges were allowed when an approval was given, apparently not the case. I merged it now, thank you for your contribution!

**Pull request merged**